app/concepts/tickets/use_cases/seat_available.rb

```diff
@@ -0,0 +1,37 @@
+module Tickets
+  module UseCases
+    class SeatAvailable
```

Comment:
This probably should work, but we can be sure if you test that. It would be great if you added some Rspec tests for that class. I'm also wondering about the `available_seats` column in the `screenings` table. What is the purpose of that column? Maybe you could use it here?

Reply:
I created this column `available_seats` earlier when I had a different approach . Right now is just copy of `cinema_hall.seats` . I think I don't need it anymore but I'm still figuring it out.